src/app/modules/main/chat_section/module.nim

```diff
@@ -599,6 +599,8 @@ method onCommunityCategoryDeleted*(self: Module, cat: Category, chats: seq[ChatD
       newCategoryName = "",
       newCategoryPosition = -1,
     )
+  self.view.chatsModel().removeItemById("cat-" & cat.id)
```

Comment:
Need to ensure we remove ghost items to have categories properly disappear

Reply:
Nice find. Could you put `cat-` in a const for here and above so it's more clear. thanks.